types/amap-js-api-tools/ranging-tool.d.ts

```diff
@@ -0,0 +1,121 @@
+declare namespace AMap {
+    namespace RangingTool {
+        interface Options {
+            /**
+             * 设置量测起始点标记属性对象，包括点标记样式、大小等，参考 MarkerOptions
+             */
+            startMarkerOption?: Marker.Options | undefined;
+            /**
+             * 设置量测中间点标记属性对象，包括点标记样式、大小等，参考 MarkerOptions
+             */
+            midMarkerOptions?: Marker.Options | undefined;
+            /**
+             * 设置量测终点标记属性对象，包括点标记样式、大小等，参考 MarkerOptions
+             */
+            endMarkerOptions?: Marker.Options | undefined;
+            /**
+             * 设置距离量测线的属性对象，包括线样式、颜色等，参考 PolylineOptions
+             */
+            lineOptions?: Polyline.Options | undefined;
+            /**
+             * 	设置距离量测过程中临时量测线的属性对象，包括线样式、颜色，参考 PolylineOptions
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/amap-js-api-tools/ranging-tool.d.ts;
  Line: 21;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
removed redundant space